### DISCUSSION TITLE
BrokenProcessPool using Fastai

### DISCUSSION THREAD

**Discussion by rafiff23** [teamalgoritma/community#560] [Q&A]
saya sudah pernah nanya di stack overflow tetapi belum ada yang bisa jawab apakah dari team algoritma bisa bantu?

https://stackoverflow.com/questions/75519105/attributeerror-module-fastai-train-has-no-attribute-iloc?noredirect=1#comment133240915_75519105

[roberta.zip](https://github.com/teamalgoritma/community/files/10795007/roberta.zip)

**Comment by finesaaa** [comment]
Dear @rafiff23,
Terima kasih atas pertanyaannya pak, tim kami akan segera coba bantu atau carikan referensi ya pak
